### DISCUSSION TITLE
Which file should I fix to use the 8081 port?

### DISCUSSION THREAD

**Discussion by Jamong435** [OvenMediaLabs/OvenMediaEngine#1116] [Help] [answered]
I want to use rest api
Which file should I fix to use the 8081 port?

[image](https://user-images.githubusercontent.com/66460135/218412621-d201a6aa-8bd2-4317-8950-598ff2892f20.png)

<Server version="8">
	
	<Bind>
		<Managers>
			<API>
				<Port>8081</Port>
				
			</API>
		</Managers>
		
	</Bind>
	
</Server>

**Comment by getroot** [comment, marked as answer]
Modify your Server.xml by referring to the document below.
https://airensoft.gitbook.io/ovenmediaengine/configuration

**Comment by Jamong435** [comment]
Thanks to you, I can use API now

{
    "message": "[HTTP] Authorization header is required to call API (403)",
    "statusCode": 403
}

How do I set up the Authorization header?

**Comment by getroot** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication

Use "Basic" authentication scheme

**Comment by Jamong435** [comment]
sorry ,
How do I activate the controller?
{
    "message": "[HTTP] Controller not found (404)",
    "statusCode": 404
}

**Comment by getroot** [comment]
It's because you accessed a non-existent resource (404 error).

**Please read the manual.**

https://airensoft.gitbook.io/ovenmediaengine/rest-api

**Comment by umeshshende** [comment]
@Jamong435  use http://url:8081/v1/stats/current/vhosts/default i got the same error 404 but then realise use /v1/stats/current/vhosts/default
